lib/src/sql/mod.rs

```diff
@@ -1,6 +1,8 @@
 //! The full type definitions for the SurrealQL query language
 
 pub(crate) mod algorithm;
+#[cfg(feature = "arbitrary")]
+pub mod arbitrary;
```

Comment:
Done. Although it's worth pointing out that the fuzz target needs public access (because it's in it's own fuzzing specific crate) to the arbitrary implementations, and surprisingly it still builds fine. So I assume that because the trait is public and in a foreign crate that the visibility definitions have no effect. 

> I think it may be useful for others in the future, but since we have taken some fuzzing-oriented decisions (e.g. sleep statement is always zero), I would prefer third-parties to not rely on it for the time being.

Just as a note the `arbitrary` crate was specifically created for the purpose of making fuzzing easier and is [owned by the rust-fuzz working group](https://github.com/rust-fuzz/arbitrary/). I haven't really seen it used for anything other than fuzzing.

> I would prefer third-parties to not rely on it for the time being.

I still think that this is a reasonable, but we might need to do this in a different way. Perhaps I could rename the optional feature from "arbitrary"->"internal_use_only_fuzzing". It'd still be possible for a third_party to enable this feature but they'd do so with the explicit understanding that the feature isn't explicitly supported.

Reply:
I think the name of the feature is very apt. If the module needs to be public, it would also be okay to preface it with `#[doc(hidden)]`, I just suggested making it internal to the crate because I also tested it and it seemed to build fine 😅